.travis.yml

```diff
@@ -1,16 +1,20 @@
 language: python
-sudo: false
+
 python:
   - 2.7
   - 3.4
   - 3.5
   - 3.6
-  - pypy-5.4.1
+  - 3.7
+  - 3.8-dev
+  - pypy3
```

Comment:
Why no `pypy`, just `pypy3`?

Reply:
I have zero experience (and interest in pypy), so I just googled what pypy-5.4.1 gets replaced with and it seemed to me it is pypy3.

Added pypy, too.